portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,58 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
```

Comment:
I would prefer you actually made a separate slideshow.js script that would contain a SlideShow class so that you dont have to keep using global variables but i wont make you do it this time. But in the future, you wont to try and use abstractions and object oriented programming when applicable. You dont want to just dump stuff in global